lib/internal/test_runner/test.js

```diff
@@ -746,20 +765,20 @@ class Test extends AsyncResource {
     this.finished = true;
   }
 
-  #duration() {
+  duration() {
     // Duration is recorded in BigInt nanoseconds. Convert to milliseconds.
     return Number(this.endTime - this.startTime) / 1_000_000;
```

Comment:
should this access a private field so it can only be used on proper instances?

Reply:
No it's fine. The `Test` class and the classes that extend it are not supposed to be accessible from userland. If they are, it's a bug. I needed this change for this PR.